### PR TITLE
UI

### PR DESCRIPTION
**Pull request by emilyvera**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,19 +7,133 @@
...
+                <img src="/images/headshot.jpg" alt="My headshot" width="400" height="500" class="real-headshot"> 
+            </div>
+            <div class="two-column-right">
+                <h2 class="aboutme-block">Hi, I'm Emily. Nice to meet you.</h2>
+                <p class="white aboutme-block">My name is Emily Vera and I am a rising junior studying 
+                computer science at the University of Illinois at Urbana-Champaign. In addition to 
+                computers, I am also heavily interested in entreprenuership and the business side of 
+                things, and am minoring in technology and management. I am in a social sorority: 
+                Phi Sigma Sigma, and am also involved in other clubs such as Society of Women Engineers, 
+                Girls Who Code, and am currently a STEP Intern at Google!</p>
```
Comment (bryyeh) [inline portfolio/src/main/webapp/index.html:37]: You can take out optional tags. I believe the &lt;/p&gt; tag is optional.

https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,19 +7,133 @@
...
+            </div>
+            <div class="row">
+                <div class="five-column">
+                    <a id="socials"></a>
```
Comment (bryyeh) [inline portfolio/src/main/webapp/index.html:41]: Did you mean to put a href here?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,273 @@
...
+
+.projects {
+    border-radius: 15px;
+}
+
+.three-column-left {
+    background-color: white;
+    border-bottom-left-radius: 15px;
+    border-top-left-radius: 15px;
+    box-shadow: 0 0 1pt 0.5pt #d5d1d1;
```
Comment (bryyeh) [inline portfolio/src/main/webapp/style.css:138]: Omit leading “0”s in values. Check rest of the file for this.

https://google.github.io/styleguide/htmlcssguide.html#Leading_0s

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,273 @@
...
+}
+
+h4 { 
+    color: black;
+    font: normal normal 100 25px/21.4px Rockwell, 'Courier Bold', Courier, Georgia, Times, 'Times New Roman', serif;
+    margin-top: 30px;
+    white-space: nowrap;
+}
+
+p.white {
```
Comment (bryyeh) [inline portfolio/src/main/webapp/style.css:54]: Avoid qualifying ID and class names with type selectors.

https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (acheng1) [reply portfolio/src/main/webapp/index.html:37]: I think the linter might actually complain about not having end tag.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (emilyvera) [reply portfolio/src/main/webapp/index.html:41]: yes the navigation bar will know by the id to go to this spot

**Pull request closed without merging**